.github/workflows/codeql.yml

```diff
@@ -0,0 +1,30 @@
+name: CodeQL
```

Comment:
Please leave this out of the PR, I'm not convinced it's worth it.

Reply:
Ok